### PR TITLE
Input handler cleanup

### PR DESCRIPTION
**Pull request by amazingalek**
Just formatting/cleanup of small stuff. Larger stuff I left as comments on original PR.

### CODE REVIEW

File: OWML.ModHelper.Input/OWML.ModHelper.Input.csproj [hunk 1]
```diff
@@ -81,7 +81,6 @@
     <Compile Include="ModInputTextures.cs" />
     <Compile Include="ModInputLibrary.cs" />
     <Compile Include="Properties\AssemblyInfo.cs" />
-    <Compile Include="ModInputLibrary.cs" />
```
Comment (amazingalek) [inline OWML.ModHelper.Input/OWML.ModHelper.Input.csproj:84]: Not sure what this means, it's still there... 🤷

File: OWML.ModHelper.Menus/ModInputCombinationElement.cs [hunk 2]
```diff
@@ -49,9 +48,9 @@ override public string Title
             layoutGroup.childForceExpandHeight = false;
             layoutGroup.childForceExpandWidth = false;
             layoutGroup.spacing = 0f;
-            var constantGraphics = _layoutObject.GetComponentsInChildren<Graphic>(true);
-            _layoutObject.transform.GetComponentInChildren<Text>(true).gameObject.SetActive(false);
-            Layout = new LayoutManager(layoutGroup, MonoBehaviour.FindObjectOfType<UIStyleManager>(),
+            var constantGraphics = layoutObject.GetComponentsInChildren<Graphic>(true);
+            layoutObject.transform.GetComponentInChildren<Text>(true).gameObject.SetActive(false);
+            Layout = new LayoutManager(layoutGroup, GameObject.FindObjectOfType<UIStyleManager>(),
```
Comment (TAImatem) [inline OWML.ModHelper.Menus/ModInputCombinationElement.cs:53]: You sure those don't work differently?

File: OWML.ModHelper.Menus/LayoutManager.cs [hunk 3]
```diff
@@ -23,16 +21,17 @@ public class LayoutManager : ILayoutManager
...
+            styleApplier.SetBackground(backGraphics);
         }
 
         public LayoutManager(LayoutGroup layout, UIStyleManager styleManager, ModUIStyleApplier styleApplier,
-            Vector3 scale, Graphic[] constantGraphics) : this(layout, styleManager, styleApplier, scale)
+            Vector3 scale, Graphic[] constantGraphics)
+            : this(layout, styleManager, styleApplier, scale)
         {
-            Array.ForEach(constantGraphics, element => _constantGraphics.Add(element));
+            constantGraphics.ToList().ForEach(element => _constantGraphics.Add(element));
```
Comment (TAImatem) [inline OWML.ModHelper.Menus/LayoutManager.cs:34]: won't compile 🤷‍♂️

File: OWML.ModHelper.Input/OWML.ModHelper.Input.csproj (hunk 1 above)
Comment (TAImatem) [reply OWML.ModHelper.Input/OWML.ModHelper.Input.csproj:84]: ah, that's a random duplicate. I kinda forgot to remove it

**Pull request merged**